releasenotes/notes/fix-apm-shutdown-35851730f8c9dad5.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    APM: Fix a deadlock issue which can prevent the trace-agent from shutting down.
```

Comment:
Can this affect anyone, or is it Windows-only? Just making sure we've narrowed the scope of this release note as much as possible.

Reply:
It's possible to affect anybody, but we've only seen it happen on Windows.